### PR TITLE
Adding search to Loggly typing and updating to 1.1 [Types-2.0]

### PR DESCRIPTION
**Pull request by geoffreak**
case 2. Improvement to existing type definition.
- documentation or source code reference which provides context for the suggested changes.  url http://api.jquery.com/html .
  - it has been reviewed by a DefinitelyTyped member.

---

Updating typing of [`loggly`](https://github.com/winstonjs/node-loggly) to version 1.1, which includes the `search` functionality. Also cleaned up the definition.

### CODE REVIEW

File: loggly/index.d.ts [hunk 1]
```diff
@@ -1,23 +1,66 @@
...
-export interface LogglyOptions {
-    token: string;
-    subdomain: string;
-    tags?: string[];
-    json?: boolean;
-    host?: string;
-    auth?: {
-        username: string;
-        password: string;
+declare namespace Loggly {
```
Comment (mhegazy) [inline loggly/index.d.ts]: you do not need the namespace. just keep every thing on the top level with `export` and remove `export = ` at the end.

Comment (geoffreak) [inline loggly/index.d.ts]: Done 👍

**Comment by mhegazy** [comment]
thanks. waiting on the CI results.

**Comment by geoffreak** [comment]
Looks like some other typing caused CI to fail?

**Comment by mhegazy** [comment]
I have a fix for this in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/11719

**Comment by geoffreak** [comment]
I've rebased this branch on your fix

**Pull request merged**